fullstop/cli.py

```diff
@@ -221,6 +229,8 @@ def list_violations(config, output, since, region, meta, remeta, limit, all, **k
 
     params = {'size': limit, 'sort': 'id,DESC'}
     params['from'] = parse_since(since)
+    params['application-ids'] = kwargs.get('applications')
```

Comment:
needed to explicitly map those params, because Fullstop API requires dashesin the names and `click.option` creates parameter names with underscores automatically.